src/Panel/VariablesPanel.php

```diff
@@ -103,7 +129,23 @@ public function shutdown(Event $event)
                 );
             } elseif (is_object($item) && method_exists($item, '__debugInfo')) {
                 // Convert objects into using __debugInfo.
-                $item = array_map($walker, $item->__debugInfo());
+                $item = $this->_walkDebugInfo($walker, $item);
+            }
+
+            if (is_object($item) ||
+                is_resource($item)
```

Comment:
Resources aren't generally serializable. Perhaps we should just print the resource type instead?

Reply:
Good point, I mixed things up, showing that it's a resource alongside with its type sounds like a good idea.